sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -3,35 +3,43 @@
-- Added the following enums: `AgeUnit`, `AreaUnit`, `DateTimeSubKind`, `InformationUnit`,`LengthUnit`, `NumberKind`, `RangeKind`, `RelativeTo`, `SpeedUnit`, `TemperatureUnit`,`TemporalModifier`, `VolumeUnit`, and `WeightUnit`.
-- Added the `AbstractSummaryResult.DetectedLanguage` property.
-- Added the `AnalyzeHealthcareEntitiesResult.DetectedLanguage` property.
-- Added the `AnalyzeSentimentResult.DetectedLanguage` property.
-- Added the `ClassifyDocumentResult.DetectedLanguage` property.
-- Added the `ExtractKeyPhrasesResult.DetectedLanguage` property.
-- Added the `ExtractSummaryResult.DetectedLanguage` property.
-- Added the `RecognizeEntitiesResult.DetectedLanguage` property.
-- Added the `RecognizeLinkedEntitiesResult.DetectedLanguage` property.
-- Added the `RecognizePiiEntitiesResult.DetectedLanguage` property.
+- Added support for extractive summarization.
+  - Added the `TextAnalyticsActions.ExtractSummaryActions` property to support extractive summarization in a batch of actions.
+  - Added the `TextAnalyticsClient.StartExtractSummary` and `TextAnalyticsClient.StartExtractSummaryAsync` methods to perform extractive summarization on a collection of documents.
+- Added support for abstractive summarization.
+  - Added the `TextAnalyticsActions.AbstractSummaryActions` property to support abstractive summarization in a batch of actions.
+  - Added the `TextAnalyticsClient.StartAbstractSummary` and `TextAnalyticsClient.StartAbstractSummaryAsync` methods to perform abstractive summarization on a collection of documents.
+- Added support for Fast Healthcare Interoperability Resources (FHIR) bundles as part of healthcare text analysis.
+  - Added the `AnalyzeHealthcareEntitiesOptions.FhirVersion` and `AnalyzeHealthcareEntitiesOptions.DocumentType` properties.
+  - Added the `AnalyzeHealthcareEntitiesAction.FhirVersion` and `AnalyzeHealthcareEntitiesAction.DocumentType` properties.
+  - Added the `AnalyzeHealthcareEntitiesResult.FhirBundle` property.
+  - Added the `WellKnownFhirVersion` and `HealthcareDocumentType` enums.
+- Added support for confidence scores in healthcare entity relations.
+  - Added the `HealthcareEntityRelation.ConfidenceScore` property.
+- Added support for named entity recognition (NER) resolutions.
+  - Added the `CategorizedEntity.Resolutions` property.
+  - Added the following classes: `AgeResolution`, `AreaResolution`, `BaseResolution`, `BooleanResolution`, `CurrencyResolution`, `DateTimeResolution`, `InformationResolution`, `LengthResolution`, `NumberResolution`, `NumericRangeResolution`, `OrdinalResolution`, `SpeedResolution`, `TemperatureResolution`, `TemporalSpanResolution` `VolumeResolution`, and `WeightResolution`.
+  - Added the following enums: `AgeUnit`, `AreaUnit`, `DateTimeSubKind`, `InformationUnit`,`LengthUnit`, `NumberKind`, `RangeKind`, `RelativeTo`, `SpeedUnit`, `TemperatureUnit`,`TemporalModifier`, `VolumeUnit`, and `WeightUnit`.
+- Added support for automatic language detection.
```

Comment:
Did you add the [defaultLanguage ](https://github.com/Azure/azure-rest-api-specs/blob/610dd7998fcde00822f1f65c3354ec954fb9eede/specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json#L532) parameter to startAnalyzeActions as well?

Reply:
Thank you for catching this one! I have created an issue to track it: https://github.com/Azure/azure-sdk-for-net/issues/32738